### DISCUSSION TITLE
Опять подушим крайм

### DISCUSSION THREAD

**Discussion by delingar** [octothorp-team/public-dobrograd#711] [Предложения]
В последнее время краймюки совсем оборзели. Мне кажется, им нужно умерить пыл, пока Доброград не превратился в Cops vs Robbers
Я предлагаю несколько вариантов
(нужное выбрать хайстаффу):
1. Восстановить в правилах требование для бандитов на ношение механ маски или механ балаклавы для крайма где-либо.
2. Вернуть систему что воскрешение игрока зависит от его кармы, чтобы краймерки с минус тысячной кармой подумали лишний раз перед тем как налететь на бизнес
3. Искусственно повысить цены на оружие путём повышения цен на токарные программы, удорожание Крафта кирок, удорожание патрон(опять), удорожание Крафта оружия
4. Усилить дпд и муниципалов, чтобы было больше шансов погибнуть у краймеров

**Comment by Dawaren** [comment]
1. Зачем? Как оно поможет этой ситуации?
2. Я за, но чтобы было ограничение хотя-бы на 5 часов, иначе будет слишком большие числа.
3. Цены и так космические. Некоторые ставят цену на сталь в 4к, а ещё повысить цену уже будет слишком.
4. Против. Они и так сильные. У них есть щиты, много патрон, поддержка, а ты предлагаешь их ещё усилить?

**Comment by gasw1n** [comment]
Крайм итак задушили по сравнению с тем что было. Сейчас все более менее нормально. Я с кармой -200 ресаюсь уж оч долго. Дпд и муники тож нормик.

**Comment by SnakeLuck** [comment]
Против. 
1) А смысл от этого, это просто лишняя морока
2) Здесь согласен это помогло бы против мамкиных круймеров 228
3) Чего? как это поможет? вы такими темпами сами себя задушите
4) Бред. Муниципалы и ДПД а уже тем более СВАТ очень имбалансны, их хер убьешь, хотя муниципалов еще возможно, но ДПД это практически невозможно
